packages/Checkbox/src/Checkbox.js

```diff
@@ -41,8 +31,11 @@ const Checkbox = props => {
     size,
   };
 
+  if (extendedInputProps.refLabel) {
+    delete extendedInputProps.refLabel;
```

Comment:
What harm is there in leaving this?

Reply:
We can avoid the console warnings if we delete the prop, I don't see why we wouldn't do preemptively do that

Reply:
Ah... I have mixed feelings about that.  On the one hand, you're right, we shouldn't promote a technique that produces console warnings.  

On the other hand, they're _just warnings_ and I'm not keen on the `<Checkbox>` knowing details about the `<FormElement>` implementation.  What if the `FormElement` changes again, will anyone know to come looking in the `Checkbox` to remove or change this code?

Maybe @nahumzs or @alexzherdev have some opinions or suggestions about this detail?

Reply:
Good point, Ok sure I can remove but yep we shall get warnings and unwanted attributes being added to elements but perhaps that is acceptable 🤷‍♂️

Reply:
no strong feelings but I'm leaning towards what @mikrotron said

Reply:
Ok, I think for now i've removed those prop deletions, but in future if it becomes more of an issue we can always patch it using the propsCollector to prevent `refLabel` being added to the element.